src/options/options.css

```diff
@@ -20,51 +20,51 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-ul {
-  margin: 0px;
-  padding: 0px;
-}
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
+form {
+  margin: 16px 32px;
 }
-/* in a fieldset e.g. we use a condensed version */
-.condensed-list {
-  margin-left: 0px;
+
+h1 {
+  margin: 0;
+  padding: 16px 0;
+
+  border-top: 1px solid rgba(12, 12, 13, 0.15);
+  font-weight: 300;
+  font-size: 1.46em;
 }
 
-fieldset {
-  /* same border style as separators above */
-  border: 1px solid;
-  border-radius: 4px;
-  border-color: var(--grey-30); /* Firefox: --in-content-box-border-color */
+h1:first-child {
+  border-top: none;
+}
 
-  /* a big width, but not too big */
-  display: inline-block;
-  width: auto;
-  min-width: 60vw;
+section {
+  margin-bottom: 32px;
+}
 
+h2{
   margin-bottom: 4px;
+  font-weight: 600;
+  font-size: 1.14em;
 }
 
-/* when a setting has a label *before* it, add some margin */
-label + .setting {
-  margin-left: 8px;
+.line {
+  padding: 2px 0;
+  margin-bottom: 4px;
+
+  display: flex;
+  align-items: center;
 }
 
-.helper-text {
-  display: block;
-  color: var(--grey-50);
+.item .line * {
```

Comment:
Is this `*` selector really needed? :thinking:

Reply:
This is to add the 8px padding to all first class elements in the row, as per the firefox design.

Reply:
Okay, another image:

![grafik](https://user-images.githubusercontent.com/11966684/54418113-41bf2d00-4704-11e9-817c-294400068901.png)

Looks better this way. 😃 

That said, do you have a link where the Photon guide states this?

Reply:
Also consider to add references/some explanation to the CSS source, if needed.

Reply:
limiting the 8px margin to only elements used, is kinda like *

![](https://design.firefox.com/photon/images/grid/grid-spacing.svg)